examples/tests/human.nbt

```diff
@@ -23,7 +23,7 @@ assert_eq((1 week           -> human), "7 days")
 assert_eq((1.5 weeks        -> human), "10 days + 12 hours")
 assert_eq((2 weeks          -> human), "14 days")
 
-assert_eq((1 sidereal_day   -> human), "23 hours + 56 minutes + 4.090500 seconds")
+assert_eq((1 sidereal_day   -> human), "23 hours + 56 minutes + 4.0905 seconds")
```

Comment:
@BurntSushi `%.f` works great. I like jiffs version better than what chrono does (and I could probably get the same behavior by using `%.6f`). I just wanted to mention that it's not exactly the same (see trailing zeros for chrono).

Reply:
Oh, wow. I tried out some more cases. This program:

```rust
fn main() -> anyhow::Result<()> {
    let dt = chrono::NaiveDateTime::parse_from_str(
        "2024-07-28 14:30:01.1",
        "%F %H:%M:%S%.f",
    )?;
    let string = dt.format("%F %H:%M:%S%.f").to_string();
    println!("{string}");

    let dt = chrono::NaiveDateTime::parse_from_str(
        "2024-07-28 14:30:01.12",
        "%F %H:%M:%S%.f",
    )?;
    let string = dt.format("%F %H:%M:%S%.f").to_string();
    println!("{string}");

    let dt = chrono::NaiveDateTime::parse_from_str(
        "2024-07-28 14:30:01.123",
        "%F %H:%M:%S%.f",
    )?;
    let string = dt.format("%F %H:%M:%S%.f").to_string();
    println!("{string}");

    let dt = chrono::NaiveDateTime::parse_from_str(
        "2024-07-28 14:30:01.1234",
        "%F %H:%M:%S%.f",
    )?;
    let string = dt.format("%F %H:%M:%S%.f").to_string();
    println!("{string}");

    Ok(())
}
```

Has this output:

```
2024-07-28 14:30:01.100
2024-07-28 14:30:01.120
2024-07-28 14:30:01.123
2024-07-28 14:30:01.123400
```

I'm not sure if this is intended behavior either. It's pretty weird. It looks like it "snaps" to precision of milliseconds, microseconds or nanoseconds automatically. I don't see any place where this is documented. But yeah, Jiff doesn't behave like this (and I don't think it should).